salt/elife-dashboard/uwsgi.sls

```diff
@@ -8,6 +8,8 @@ app-nginx-conf:
             {% if pillar.elife.env != 'dev' %}
             - cmd: web-ssl-enabled
             {% endif %}
+        - listen_in:
```

Comment:
this is an nginx configuration, so it makes sense to add it here

Reply:
I think it's also needed here https://github.com/elifesciences/elife-dashboard-formula/blob/60871e6ed3667e199c9b73adac8567a6c018f28b/salt/elife-dashboard/dashboard2.sls#L46
and probably here https://github.com/elifesciences/elife-dashboard-formula/blob/master/salt/elife-dashboard/scheduler.sls#L98